internal/workload/manager.go

```diff
@@ -195,6 +211,19 @@ func (w *WorkloadManager) Update(configuration models.DeviceConfigurationMessage
 	return errors
 }
 
+func (w *WorkloadManager) manageAuthFile(authFilePath, authFile string) (string, error) {
+	if authFile == "" {
+		if err := deleteFile(authFilePath); err != nil {
+			return "", fmt.Errorf("cannot remove auth file %s: %s", authFilePath, err)
+		}
+		return "", nil
+	}
+	if err := w.storeFile(authFilePath, []byte(authFile)); err != nil {
```

Comment:
For stored files how do we handle secret updates?

Reply:
The secret gets overwritten if change is detected here: https://github.com/jakub-dzon/k4e-device-worker/pull/53/files#diff-8be2736004b01f09b6ffef40a12d93f4f3f99cf3c84bc32e78100c086122cc44R151.